src/EditorFeatures/TestUtilities/CodeActions/AbstractCodeActionTest.cs

```diff
@@ -63,9 +63,9 @@ protected override Task<ImmutableArray<Diagnostic>> GetDiagnosticsWorkerAsync(Te
             var documentsWithSelections = workspace.Documents.Where(d => !d.IsLinkFile && d.SelectedSpans.Count == 1);
             Debug.Assert(documentsWithSelections.Count() == 1, "One document must have a single span annotation");
             var span = documentsWithSelections.Single().SelectedSpans.Single();
-            var actions = ArrayBuilder<CodeAction>.GetInstance();
+            var actions = ArrayBuilder<(CodeAction, TextSpan?)>.GetInstance();
             var document = workspace.CurrentSolution.GetDocument(documentsWithSelections.Single().Id);
-            var context = new CodeRefactoringContext(document, span, actions.Add, CancellationToken.None);
+            var context = new CodeRefactoringContext(document, span, (a, t) => actions.Add((a, t)), CancellationToken.None);
```

Comment:
does `actions.Add` not work here?

Reply:
Because it's two parameters and not tuple. And C# can't automatically apply two parameters as tuple (inverse of deconstruction).